ipetcare.spa/src/features/race/components/raceForm.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import { BASE_URL } from '../../../utils/constants'
+import '../race.css'
+import {
+  FormControl,
+  FormLabel,
+  Button,
+  TextField,
+  FormHelperText,
+  Select,
+  MenuItem,
+  InputLabel,
+} from '@material-ui/core'
+import { getUserState } from '../../../utils/localStorageHelper'
+import { useParams } from 'react-router-dom'
+
+axios.interceptors.request.use(
```

Comment:
Nw czy to miało być na tym branchu ale to chyba trzeba refaktornąć. Chyba, że to już zostawiamy dla @blc132